src/dagmc.cpp

```diff
@@ -268,14 +268,12 @@ void load_dagmc_geometry()
     std::string cmp_str = mat_value;
     to_lower(cmp_str);
 
-    if (cmp_str.find("graveyard") != std::string::npos) {
+    if (cmp_str == "graveyard") {
```

Comment:
dumb question is cmp_str to_lower() at some point? so Graveyard and Vacuum would count too?

Reply:
I have tried Vacuum_vessel and vacuum_vessel and both cases work fine for me.